Server/src/RedisManager.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2024 Microsoft, Inc. and others.  All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ */
+
+#include "RedisManager.h"
+
+
+/*********************************************************************//**
+ * Constructor for class
+ ***********************************************************************/
+RedisManager::RedisManager() {
```

Comment:
>RedisManager [](http://example.com/codeflow?start=14&length=12)

Any thread safe protection for all methods?

Reply:
This will be single thread used now, only one connection will be triggered from FRR to BMP, for multiple asic case FRR/BMP should also be pair under the same namespace docker. Thus now the workflow is like below:

1. openbmpd starts, FRR connects to openbmpd
2. openbmpd start populating BMP_STATE_DB
3. once any CONFIG_DB change, bmpcfgd https://github.com/sonic-net/sonic-buildimage/pull/18940 will monitor and restart openbmpd for a refresh.